### DISCUSSION TITLE
how clean my predicate

### DISCUSSION THREAD

**Discussion by aymenbraiek** [Netflix/dgs-framework#1535] [General]
Hello guys :
I have my query DGS findById(Predicate filter  )
I Define My predicate 
if test my query without put some field of my filter 
this error is display
Exception while fetching data (/fd) : org.hibernate.hql.internal.ast.QuerySyntaxException: Invalid path:
Any suggest or solution

**Comment by coleturner** [comment]
Hi @aymenbraiek there is not enough info here to be able to help. Please provide a full stack trace or a repro link. The exception you shared does not seem to relate to DGS
